scripts/proposals/proposal_04_CM_guard.js

```diff
@@ -0,0 +1,43 @@
+/*global process*/
+
+const { ethers } = require("hardhat");
+
+async function main() {
+    const fs = require("fs");
+    const globalsFile = "globals.json";
+    const dataFromJSON = fs.readFileSync(globalsFile, "utf8");
+    let parsedData = JSON.parse(dataFromJSON);
+
+    const cancellerRole = ethers.utils.id("CANCELLER_ROLE");
+    const timelockAddress = parsedData.timelockAddress;
+    const treasuryAddress = parsedData.treasuryAddress;
+    const guardCMAddress = parsedData.guardCMAddress;
+    const CMAddress = parsedData.CM;
+
+    // Obtaining proposal values
+    console.log("Revoking cancellor role of CM in the Timelock and enabling selectors");
+    const timelock = await ethers.getContractAt("Timelock", timelockAddress);
+    const guardCM = await ethers.getContractAt("GuardCM", guardCMAddress);
+    const targets = [guardCMAddress, timelockAddress];
+    const values = new Array(2).fill(0);
+    const callDatas = [
+        // CM is always able to schedule pause() and disableToken(token) function calls via the timelock
+        // When the new owner address are known, add the CM.swapOwner method
```

Comment:
Not sure about this one. The CM itself can't perform actions on itself after setting a guard. The timelock is able to change / swap owners via the governance vote.